JustSaying.Sample.Restaurant.KitchenConsole/Program.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Threading.Tasks;
+using Amazon;
+using JustSaying.Sample.Restaurant.Models;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace JustSaying.Sample.Restaurant.KitchenConsole
+{
+    internal class Program
+    {
+        public static async Task Main()
+        {
+            Console.Title = "KitchenConsole";
+
+            await new HostBuilder()
+                .ConfigureAppConfiguration((hostContext, config) =>
+                {
+                    config.SetBasePath(AppContext.BaseDirectory);
+                    config.AddJsonFile("appsettings.json", optional: false);
+                    config.AddJsonFile($"appsettings.{hostContext.HostingEnvironment.EnvironmentName}.json", optional: true);
+                    config.AddEnvironmentVariables();
+                })
+               .ConfigureLogging((hostContext, logging) =>
+               {
+                   logging.AddConsole();
+               })
+               .ConfigureServices((hostContext, services) =>
+               {
+                   var awsRegion = hostContext.Configuration.GetValue<string>("AWSRegion");
```

Comment:
Could just use `hostContext.Configuration["AWSRegion"];` etc. here as we're just using strings.

Reply:
fixed